app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryHistoryLocation.js

```diff
@@ -1,61 +1,68 @@
 import React from 'react';
 import Clipboard from 'react-clipboard.js';
 import { storyUrl } from '../StoryUrl';
-import { editingStoryPropTypesShape } from '../../../models/beta/story';
+import { editingStoryPropTypesShape, isRelease } from '../../../models/beta/story';
+import PropTypes from 'prop-types';
 
-class ExpandedStoryHistoryLocation extends React.Component {
-  render() {
-    const { story, onClone, showHistory } = this.props;
+const ExpandedStoryHistoryLocation = ({
+  story,
+  onClone,
+  showHistory
+}) => (
+  <div className="col-xs-12 form-group input-group input-group-sm">
+    <input
+      className="form-control input-sm"
+      id={`story-link-${story.id}`}
+      readOnly
+      value={storyUrl(story)}
+    />
 
-    return (
-      <div className="col-xs-12 form-group input-group input-group-sm">
-        <input
-          className="form-control input-sm"
-          id={`story-link-${story.id}`}
-          readOnly
-          value={storyUrl(story)}
-        />
+    <span className="input-group-btn">
+      <Clipboard
+        data-clipboard-text={storyUrl(story)}
+        component="button"
+        className="btn btn-default btn-clipboard"
+        button-title={I18n.t('story.events.copy_url')}
+      >
+        <img src="/clippy.svg" alt={I18n.t('story.events.copy_url')} className="clipboard-icon" />
+      </Clipboard>
 
-        <span className="input-group-btn">
-          <Clipboard
-            data-clipboard-text={storyUrl(story)}
-            component="button"
-            className="btn btn-default btn-clipboard"
-            button-title={I18n.t('story.events.copy_url')}
-          >
-            <img src="/clippy.svg" alt={I18n.t('story.events.copy_url')} className="clipboard-icon" />
-          </Clipboard>
+      <Clipboard
+        data-clipboard-text={`#${story.id}`}
+        component="button"
+        className="btn btn-default btn-clipboard-id btn-clipboard"
+        button-title={I18n.t('story.events.copy_id')}
+      >
+        ID
+      </Clipboard>
 
-          <Clipboard
-            data-clipboard-text={`#${story.id}`}
-            component="button"
-            className="btn btn-default btn-clipboard-id btn-clipboard"
-            button-title={I18n.t('story.events.copy_id')}
-          >
-            ID
-          </Clipboard>
+      {
+        !isRelease(story) && (
```

Comment:
What do you think about creating a function called `hasHistory(story)` that uses `isRelease` inside of it and use that new one here?